lib/datasource/pod/index.ts

```diff
@@ -34,12 +38,14 @@ function releasesGithubUrl(
 async function makeRequest<T = unknown>(
   url: string,
   lookupName: string,
-  json = true
+  fromGithub = true
 ): Promise<T | null> {
   try {
-    const resp = await api.get(url, { json });
+    const resp = fromGithub
+      ? await githubHttp.getJson<T>(url)
+      : await http.get(url);
```

Comment:
I think this is wrong, because `http.get` will return a `string` body, so line 80 does not work as expected `resp.map(({ name }) => ({ version: name }))`, because `typeof resp === 'string'`

It makes no sense to cast `resp.body` to T if it's  a string.

Reply:
Agree, it's weird place... I'll figure it out with more of your comments as well.